### PR TITLE
Convert CommonJS module syntax to ES6

### PR DESCRIPTION
**Pull request by danielrearden**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -321,9 +321,9 @@ running a GraphQL request. This function is used internally to handle the
 incoming request, you may use it directly for building other similar services.
 
 ```js
-const graphqlHTTP = require('express-graphql');
+const { graphqlHTTP } = require('express-graphql');
```
Comment (IvanGoncharov) [inline README.md:324]:
```suggestion
const { graphqlHTTP, getGraphQLParams } = require('express-graphql');
```

File: types/tsconfig.json [hunk 2]
```diff
@@ -8,6 +8,6 @@
     "strictFunctionTypes": true,
     "noEmit": true,
     "baseUrl": ".",
-    "paths": { "express-graphql": ["."] }
+    "paths": { "express-graphql": ["./index"] }
```
Comment (IvanGoncharov) [inline types/tsconfig.json:11]: Why did you change that?

File: types/index.d.ts [hunk 3]
```diff
@@ -16,179 +16,174 @@ import {
   GraphQLTypeResolver,
 } from 'graphql';
 
-export = graphqlHTTP;
+export {};
```
Comment (IvanGoncharov) [inline types/index.d.ts:19]: Why do we need this construct?

File: types/index.d.ts [hunk 4]
```diff
@@ -16,179 +16,174 @@ import {
...
 
 type Request = IncomingMessage;
 type Response = ServerResponse & { json?: (data: unknown) => void };
 
-declare namespace graphqlHTTP {
+/**
+ * Used to configure the graphqlHTTP middleware by providing a schema
```
Comment (IvanGoncharov) [inline types/index.d.ts:25]: So it just indentation changes you didn't change anything inside, right?

Comment (danielrearden) [reply types/index.d.ts:25]: Removed the namespace and exported the individual types

File: types/tsconfig.json (hunk 2 above)
Comment (danielrearden) [reply types/tsconfig.json:11]: Dangit, sorry missed that

File: types/index.d.ts (hunk 3 above)
Comment (danielrearden) [reply types/index.d.ts:19]: dtslint was complaining about the types we don't export

> All declarations in this module are exported automatically. Prefer to explicitly write 'export' for clarity. If you have a good reason not to export this declaration, add 'export {}' to the module to shut off automatic exporting. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md

File: types/tsconfig.json (hunk 2 above)
Comment (IvanGoncharov) [reply types/tsconfig.json:11]: @danielrearden No problem, it happens, e.g. yesterday I committed `console.log` into `graphql-js` 🤦

File: types/index.d.ts (hunk 3 above)
Comment (IvanGoncharov) [reply types/index.d.ts:19]: Wow, now I understand why it was complaining in `graphql-js`.
